packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -50,9 +53,10 @@ import { ResourceFetchResult } from '@internal/chat-stateful-client';
 import { AttachmentOptions } from '@internal/react-components';
 import { SendBox } from '@internal/react-components';
 /* @conditional-compile-remove(attachment-upload) */
-import { useSelector } from './hooks/useSelector';
+import { nanoid } from 'nanoid';
```

Comment:
It looks like in some other places we use `import { v1 as generateGUID } from 'uuid';`, just wanted to clarify if there is any advantages/disadvantages between them

Reply:
i used whatever that has been added in package.json, i tried to use nanoid for the other package, but it's not defined, so i use whatever that has been imported to not increase package size

Reply:
It looks like `uuid` package is in composites package.json file too
![image](https://github.com/Azure/communication-ui-library/assets/98852890/f1b22c4b-723f-4ae8-9e9e-75e4f9204ee9)

Reply:
are you sure? 
<img width="818" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/80186b88-e80a-4872-9a4c-18cf7a5b882c">

Reply:
sorry to mention the line I added `import { v1 as generateGUID } from 'uuid'` is from a test under component, which doesn't have nano id package added